public/components/integrations/components/integration_overview_panel.tsx

```diff
@@ -4,65 +4,76 @@
 
-const pageStyles: CSS.Properties = {
+const newNavigation = coreRefs.chrome?.navGroup.getNavGroupEnabled();
+
+const pageStyles = {
   width: '100%',
-  justifyContent: 'spaceBetween',
 };
 
 export function IntegrationOverview(props: any) {
   const config = props.integration;
-  return (
-    <EuiPageHeader
-      style={{ justifyContent: 'spaceBetween' }}
-      data-test-subj={`${config.name}-overview`}
-    >
-      <EuiSpacer size="m" />
+
+  const buttons = (
+    <>
+      <EuiFlexItem grow={false}>
+        <EuiSmallButton
+          onClick={() => {
+            props.setUpSample();
+          }}
+          disabled={props.loading}
+          data-test-subj="try-it-button"
+          data-click-metric-element="integrations.create_from_try_it"
+        >
+          Try It
+        </EuiSmallButton>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiSmallButton
+          onClick={() => {
+            props.showFlyout(config.name);
+          }}
+          fill
+          disabled={props.loading}
+          data-test-subj="add-integration-button"
+          data-click-metric-element="integrations.set_up"
+        >
+          Set up integration
+        </EuiSmallButton>
+      </EuiFlexItem>
+    </>
+  );
+
+  return newNavigation ? (
+    <HeaderControlledComponentsWrapper
+      components={[
+        <EuiFlexGroup gutterSize="s" alignItems="flexEnd" responsive={false}>
+          {buttons}
+        </EuiFlexGroup>,
+      ]}
+    />
+  ) : (
+    <EuiPageHeader data-test-subj={`${config.name}-overview`}>
       <EuiPageHeaderSection style={pageStyles}>
         <EuiPageContentHeaderSection>
           <EuiFlexGroup gutterSize="xs" justifyContent="spaceBetween">
             <EuiFlexItem>
-              <EuiText data-test-subj="eventHomePageTitle" size="s">
-                <h1>{config.displayName || config.name}</h1>
-              </EuiText>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButton
-                size="m"
-                onClick={() => {
-                  props.setUpSample();
-                }}
-                disabled={props.loading}
-                data-test-subj="try-it-button"
-                data-click-metric-element="integrations.create_from_try_it"
-              >
-                Try It
-              </EuiButton>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButton
-                size="m"
-                onClick={() => {
-                  props.showFlyout(config.name);
-                }}
-                fill
-                disabled={props.loading}
-                data-test-subj="add-integration-button"
-                data-click-metric-element="integrations.set_up"
-              >
-                Set Up
-              </EuiButton>
+              <EuiTitle data-test-subj="eventHomePageTitle" size="l">
+                <h3>{config.displayName || config.name}</h3>
```

Comment:
I see we're meant to be prioritizing the display name here (this is correct), but in the screenshot it's falling back to the non-display name. Can you check why/fix it? ![](https://github.com/user-attachments/assets/921008c4-1cb6-4f14-b716-e3ce3bcde718)

Reply:
While under the new navigation the headers are generated by dashboards directly. They pull from the breadcrumbs and display it as the title while removing the breadcrumb.
This part is rendered by the following code:
```
  return newNavigation ? (
    <HeaderControlledComponentsWrapper
      components={[
        <EuiFlexGroup gutterSize="s" alignItems="flexEnd" responsive={false}>
          {buttons}
        </EuiFlexGroup>,
      ]}
    />
```
The only way to change this would be to update the breadcrumb to be the display name. Not sure if we want that change though, or how it wold effect the old navigation.

Reply:
Ah, got it, so it's displaying the URL path instead of the title -- I'm not sure if I like that (since it's what the display name is for and the integ IDs are not necessarily always readable) but if it's inconvenient to change then I guess we can keep it?

Reply:
I see in the [breadcrumbs docs](https://oui.opensearch.org/1.12/#/navigation/breadcrumbs) that it's possible to have separate `href` and `text`, is that applicable here at all?